docs/create-packages/Symbol-Packages-V2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to publish NuGet symbol packages using the new symbol package format '.snupkg'| Microsoft Docs
+author:
+- cristinamanu
+- kraigb
+ms.author:
+- cristinamanu
+- kraigb
+manager: skofman
+ms.date: 10/30/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: How to create NuGet packages that contain only symbols to support debugging of other NuGet packages in Visual Studio.
+keywords: NuGet symbol packages, NuGet package debugging, supporting NuGet debugging, package symbols, symbol package conventions
+ms.reviewer:
+- anangaur
+- karann
+- ---
+
+
+# Creating symbol packages (.snupkg)
+> [!Important]
+> This feature is available only with [nuget.exe v4.9.0 or above](https://www.nuget.org/downloads) or [dotnet.exe v2.2.0 or above](https://www.microsoft.com/net/download/dotnet-core/2.2), which implement the required [NuGet protocols](../api/nuget-protocols.md).
+> The new format for the symbol packages in .snupkg. The previous format .symbols.nupkg is still supported but only for compatibility reasons (see [Symbol-Packages](Symbol-Packages.md)).
+
+Nuget.org supports its own symbols server repository. In addition to building packages for nuget.org or other sources, NuGet also supports creating associated symbol packages and publishing them to the symbol server repository.
+Package consumers can use the symbols published to nuget.org symbol server by adding `https://symbols.nuget.org/download/symbols` to their symbol sources in Visual Studio, which allows stepping into package code in the Visual Studio debugger. See [Specify symbol (.pdb) and source files in the Visual Studio debugger](/visualstudio/debugger/specify-symbol-dot-pdb-and-source-files-in-the-visual-studio-debugger) for details on that process.
+
+Nuget.exe pack and msbuild /t:pack has an additional command line property called ```-SymbolsPackageFormat```. Since the symbols package will not be produced by default (as per current behavior), this option will only be applicable if passed along with ```-Symbols``` in nuget.exe or ```--include-symbols``` in dotnet.exe. SymbolsPackageFormat property can have one of the two values: snupkg, symbols.nupkg. The default value of this property would be snupkg.
+The option symbols.nupkg is only being provided for compat reasons with the current behavior. Using this in conjunction with -Symbols or --include-symbols will ensure the symbols.nupkg is created in the same way as it is today.
```

Comment:
What option is `symbols.nupkg`. Is it `-symbols.nupkg`?

Reply:
It is either the default :
nuget.exe pack ***  -Symbols 
or
nuget.exe pack *** -Symbols -SymbolPackageFormat symbols.nupkg

Updated the text.